.github/workflows/conda-package.yml

```
@@ -170,7 +170,7 @@ jobs:
       # We want to make sure that all dependecies install automatically.
       # intel::intel-opencl-rt is needed for set-intel-ocl-icd-registry.ps1
       - name: Install built package
-        run: conda install ${{ env.PACKAGE_NAME }}=${{ env.PACKAGE_VERSION }} intel::intel-opencl-rt -c ${{ env.CHANNEL_PATH }}
+        run: conda install ${{ env.PACKAGE_NAME }}=${{ env.PACKAGE_VERSION }} https://software.repos.intel.com/python/conda::intel-opencl-rt -c ${{ env.CHANNEL_PATH }} --override-channels
```

```suggestion
        run: conda install ${{ env.PACKAGE_NAME }}=${{ env.PACKAGE_VERSION }} 'https://software.repos.intel.com/python/conda::intel-opencl-rt' -c ${{ env.CHANNEL_PATH }}
```
Try this. I guess bash may split arguments...

---

No, it did not work

---

```suggestion
        run: conda install ${{ env.PACKAGE_NAME }}=${{ env.PACKAGE_VERSION }} -c ${{ env.CHANNEL_PATH }}
```
This may work

---

There is a cf repack, so it must work now: https://anaconda.org/conda-forge/intel-opencl-rt